src/main/java/uk/gov/hmcts/dts/fact/services/admin/list/AdminLocalAuthorityService.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.dts.fact.services.admin.list;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.dts.fact.exception.DuplicatedListItemException;
+import uk.gov.hmcts.dts.fact.exception.NotFoundException;
+import uk.gov.hmcts.dts.fact.model.admin.LocalAuthority;
+import uk.gov.hmcts.dts.fact.repositories.LocalAuthorityRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class AdminLocalAuthorityService {
+
+    private final LocalAuthorityRepository localAuthorityRepository;
+
+    @Autowired
+    public AdminLocalAuthorityService(final LocalAuthorityRepository localAuthorityRepository) {
+        this.localAuthorityRepository = localAuthorityRepository;
+    }
+
+    public List<LocalAuthority> getAllLocalAuthorities() {
+        return localAuthorityRepository.findAll()
+            .stream()
+            .map(LocalAuthority::new)
+            .collect(toList());
+    }
+
+    @Transactional(rollbackFor = {RuntimeException.class})
```

Comment:
I think RunTimeException is used for the rollback by default. so can simply use @Transactional

Reply:
Thanks - I missed this in the docs. Updated and removed 'rollbackFor'.